mothur_otu.bat

```diff
@@ -71,3 +71,20 @@ unique.seqs(fasta=~/Desktop/SRP018246/~/Desktop/SRP018246.fasta)
 # inspect unique sequences
 # input: unique.fasta file, output: unique.summary file
 summary.seqs(fasta=~/Desktop/SRP018246/~/Desktop/SRP018246.unique.fasta, name=~/Desktop/SRP018246/~/Desktop/SRP018246.names)
+
+#download version 128 full length silva reference sequences and seed silva reference file and rename files
+system(cp Silva.nr_128 FullLengthSilva.fasta)
+system(cp Silva.seed_128 SilvaSeed.fasta)
+
+#trim sequences 

```

Comment:
Let's go ahead and remove this last section (everything from copying the Silva library above, to the end of this current script) and create a new script to correctly format the Silva reference.